acceptance/configurations_test.go

```diff
@@ -142,6 +161,48 @@ var _ = Describe("Configurations", LConfiguration, func() {
 			Expect(err).To(HaveOccurred(), out)
 			Expect(out).To(ContainSubstring("name must consist of lower case alphanumeric"))
 		})
+
+		It("fails for missing arguments, not enough, no files", func() {
+			out, err := env.Epinio("", "configuration", "create")
+			Expect(err).To(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Not enough arguments, expected name"))
+		})
+
+		It("fails for missing arguments, not enough, with files", func() {
+			out, err := env.Epinio("", "configuration", "create", "--from-file", "dummy")
+			Expect(err).To(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Not enough arguments, expected name"))
+		})
+
+		It("fails for missing arguments, key without value", func() {
+			out, err := env.Epinio("", "configuration", "create", "foo", "a")
+			Expect(err).To(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Last Key has no value"))
+		})
+
+		It("fails for a missing path", func() {
+			out, err := env.Epinio("", "configuration", "create", "foo", "--from-file", "MISSING")
+			Expect(err).To(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("filesystem error: open MISSING: no such file or directory"))
+		})
+
+		Describe("directory", func() {
+			BeforeEach(func() {
+				err := os.MkdirAll("DIR", 0755)
```

Comment:
It's probably better to use `os.MkdirTemp("", "DIR")`, but since test are passing I'm approving this. :smile: